asciidoctorj-core/src/main/java/org/asciidoctor/ast/impl/CellImpl.java

```diff
@@ -70,7 +70,10 @@ public void setVerticalAlignment(Table.VerticalAlignment valign) {
 
     @Override
     public Document getInnerDocument() {
-        return (Document) NodeConverter.createASTNode(getRubyProperty("inner_document"));
+        IRubyObject innerDocument = getRubyProperty("inner_document");
+        if (innerDocument.isNil())
+        	return null;
+		return (Document) NodeConverter.createASTNode(innerDocument);
```

Comment:
Gotcha! :-)
Could you please add braces around the `return null`.

I probably automatically put them in there mentally around both returns that have the same indentation.